activate

```diff
@@ -72,15 +76,16 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 live=false
 
 # report in
-if ! out=$(wget -O - -q "$SCRIPT?image=$image&release=$release&vendor=$vendor&product=$product&serial=$serial&live=$live"); then
-    echo "wget failed"
+URL="${SCRIPT}?image=${image}&release=${release}&vendor=${vendor}&product=${product}&serial=${serial}&live=${live}"
+if ! out=$(wget -O - -q ${URL}); then
```

Comment:
actually I lied, I can't do this - I can't capture the output and the exit state without || true because of set -e, so I have to leave it how it was